vertica/assets/configuration/spec.yaml

```
@@ -32,6 +32,28 @@ files:
             value:
               type: integer
               example: 5433
+          - name: metric_groups
```

```suggestion
          - name: metric_groups
             enabled: true
```

This option should be enabled

---

No need as currently implemented